include/termcolor/termcolor.hpp

```diff
@@ -9,9 +9,17 @@
 //! :license: BSD, see LICENSE for details
 //!
 
+#pragma once
+
 #ifndef TERMCOLOR_HPP_
 #define TERMCOLOR_HPP_
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable: 4800) //about force cast int to bool
```

Comment:
@catinwarmhands it would be really cool if you check my assumption above. If `static_cast<>` fixes the warning, then we should go that way.